example/package.json

```diff
@@ -10,16 +10,14 @@
   },
   "dependencies": {
     "@react-native-async-storage/async-storage": "^1.19.3",
-    "@react-native-community/netinfo": "^9.4.1",
     "react": "18.2.0",
     "react-native": "0.72.4",
-    "react-native-create-thumbnail": "1.6.0",
+    "react-native-create-thumbnail": "^2.0.0",
     "react-native-image-picker": "^7.0.0",
-    "react-native-paper": "^5.10.6",
     "react-native-safe-area-context": "^4.7.2",
     "react-native-screens": "^3.25.0",
-    "react-native-svg": "13.4.0",
-    "react-native-video": "^5.2.1"
+    "react-native-svg": "^14.1.0",
+    "react-native-video": "^6.0.0-beta.5"
```

Comment:
Is it possible to use stable version instead ?

Reply:
We've beed using stable version 5.2.1 and it does not support exo player and media3  players for android. 
The result is video loading so slow. It takes more than 10 seconds for 3s video. 

This version 6 is using exo player and media3 player. It loads video so fast under 1sec for any video. 
I checked the issues and pull request on this repo. Even it is beta. This works well.